tests/queryset/test_queryset.py

```diff
@@ -945,6 +945,12 @@ class Blog(Document):
             Blog.objects.insert(Blog(title=blog2.title))
 
         self.assertEqual(Blog.objects.count(), 2)
+        # pass ordered=False
```

Comment:
this test method is already very long, I'd suggest to create a new test with a clear purpose